pages/apim/3.x/event-native/apim-event-native-api-management-overview.adoc

```diff
@@ -0,0 +1,89 @@
+* Proper isolation between platform-level policies and API-level policies execution. This ensures that platform-level policies are always executed before any API-level policies during the request stage, and after any API-level policies during the response stage.
+* Removal of the need to define a scope for policies (`onRequest`, `onRequestContent`, `onResponse`, `onResponseContent`).
+* The introduction of new API types to better differentiate REST APIs from Async APIs (Websocket, SSE, Webhook), as a foundation to fully support the execution of AsyncAPI.
+
+The BETA release does not provide UI support for these capabilities.
+
+NOTE: The event-native policy execution engine mode has been released as a BETA version - it is not recommended using it in production until it is released as a GA version at the end of 2022. The GA release will come with additional features - contact us if you want to know more about it and what new features we are bringing to API Management regarding Sync & Async APIs.
+
+== BETA Features
+
+The following features are available in the BETA release.
+
+=== HTTP POST entrypoint
+
+The HTTP POST entrypoint is an Community Edition / OSS feature that enables external clients to publish data via standard HTTP POST requests and ease the ingestion of data into evented backends like Kafka.
+
+With this workflow, a client application running on HTTP can post content to the Gravitee Gateway. The Gateway can then take that content and push it over to Kafka as a message that Kafka can ingest. The Gravitee Gateway acts as a protocol mediation layer in this context. This enables API owners to quickly set up a sync-to-async integration.
+
+NOTE: The backend that you can post your content to via an HTTP POST entrypoint depends on the backend connectors that you have enabled via the link:{{ '/apim/3.x/apim_publisherguide_introducing_sme.html#what_is_sme' | relative_url }}[Service Management Ecosystem (SME)]. The BETA release uses the link:{{ '/apim/3.x/apim_publisherguide_introducing_kafka.html' | relative_url }}[Gravitee Community Edition / OSS Kafka connector]. In future releases we are planning to provide an advanced Kafka connector (as an Enterprise Edition feature) as well as support for more backend connectors such as MQTT, Kinesis, and others.
+
+The diagram below shows the workflow for this use case:
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-http-post.png %}[]
+
+=== Websocket and SSE entrypoints
+
+Support for Websocket (Community Edition/OSS) and Server-Sent Events (Enterprise Edition only) entrypoints are provided as separate plugins.
+
+NOTE: These plugins are expected shortly after the main 3.19 release and will be announced here when available.
```

Comment:
SSE is already embedded as CE entrypoint in the 3.19.

Reply:
OK reverting to old state then

Reply:
@mouligno I think there is a kind of misunderstanding here that needs to be clarify.